website/docs/reference/warehouse-profiles/trino-profile.md

```diff
@@ -42,27 +44,30 @@ trino:
   outputs:
     dev:
       type: trino
-      method: none  # optional, one of {none | ldap | kerberos}
+      method: none  # optional, one of {none | ldap | kerberos | oauth | jwt | certificate}
       user: [user]
       password: [password]  # required if method is ldap or kerberos
       database: [database name]
       host: [hostname]
       port: [port number]
       schema: [your dbt schema]
       threads: [1 or more]
+      retries: [1 or more]
       http_scheme: [http or https]
       session_properties:
-        query_max_run_time: 5d
-        exchange_compression: True
-  
-
+        [some_session_property]: [value]
```

Comment:
is there a reference to what `session_properties` are available or at least a link to the trino docs about `query_max_run_time` and other params?

Reply:
This could be a relevant link https://trino.io/docs/current/sql/set-session.html#session-properties-definition WDYT?